twins/mocks/service.go

```diff
@@ -42,18 +40,12 @@ func CreateDefinition(names []string, subtopics []string) twins.Definition {
 }
 
 // CreateSenML creates SenML record array
-func CreateSenML(n int, bn string) []senml.Record {
-	var recs []senml.Record
-	for i := 0; i < n; i++ {
-		rec := senml.Record{
-			BaseName: bn,
-			BaseTime: float64(time.Now().Unix()),
-			Time:     float64(i),
-			Value:    nil,
-		}
-		recs = append(recs, rec)
+func CreateSenML(n int, recs []senml.Record) {
+	for i, rec := range recs {
+		rec.BaseTime = float64(time.Now().Unix())
+		rec.Time = float64(i)
+		rec.Value = nil
```

Comment:
Maybe you can also use `float64(i)` here.

Reply:
I've tried many things. Problem is, rec.Value is a pointer. It's value gets somehow lost in "transition". Anyway, we don't really care for the values in these tests - we're not testing for the same values.